### PR TITLE
hw completion

### PR DESCRIPTION
**Pull request by Lyakabynka**

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -1,3 +1,66 @@
 import './style.css';
 
-console.log('Hello world');
+console.log(123123123);
+
+const btn1 = document.getElementById("btn1");
+const tt = document.getElementById("tt");
```
Comment (coolassassin) [inline main.js]: Using a one element for all tooltips is a good decision for this task, but it is not good way for making tooltip in common. It is better to create tooltip on fly if it is really used.
I guess if you will click on second button and then hover the first - first button will steal the tooltip and it is not good behaviour )

File: main.js [hunk 2]
```diff
@@ -1,3 +1,66 @@
...
+const btn2 = document.getElementById("btn2");
+
+
+btn2.addEventListener("click", (event) => {
+    if(tt.style.display == "block"){
+        tt.style.display = "none";
+        return;
+    }
+
+    const buttonRect = btn2.getBoundingClientRect();
```
Comment (coolassassin) [inline main.js]: just for note, you can destruct the object like that:
```javascript
const {top, height, left} = btn2.getBoundingClientRect();
```

File: style.css [hunk 3]
```diff
@@ -95,3 +105,89 @@ button:focus-visible {
...
+  background-color: #fff;
+  margin: 20% auto;
+  padding: 20px;
+  border-radius: 5px;
+  box-shadow: 0 4px 8px rgba(0, 0, 0, 0.2);
+  max-width: 40%;
+  position: relative;
+}
+
+.close {
```
Comment (coolassassin) [inline style.css:141]: why do you reassign `.modal`, `.modal-content`, `.close` three times?

File: style.css [hunk 4]
```diff
@@ -14,6 +14,16 @@
   -webkit-text-size-adjust: 100%;
 }
 
+.tooltip {
```
Comment (coolassassin) [inline style.css:17]: Tooltip has an arrow at the bottom
You can try to make one with ::before and ::after using these tricks:
https://css-tricks.com/snippets/css/css-triangle/

**Comment by Lyakabynka** [comment]
Implemented all required functionality.

File: index.html [hunk 5]
```diff
@@ -28,9 +28,6 @@ <h2>Modal Header</h2>
             <p>This is a modal window.</p>
         </div>
     </div>
-      <div id="tt" class="tooltip">
```
Comment (coolassassin) [inline index.html]: Fastest fix on wild west 😁

File: style.css [hunk 6]
```diff
@@ -24,6 +24,18 @@
   font-size: 14px;
 }
 
+.tooltip::after {
```
Comment (coolassassin) [inline style.css:27]: Great! 👍

File: index.html (hunk 5 above)
Comment (Lyakabynka) [reply index.html]: 🤩

**Pull request closed without merging**